pkg/backup/restore/restore_manager.go

```diff
@@ -1276,15 +1276,24 @@ func (rm *restoreManager) waitWarmUpJobsFinished(r *v1alpha1.Restore) error {
 	for _, job := range jobs {
 		jobFinished := false
 		for _, condition := range job.Status.Conditions {
-			if condition.Type == batchv1.JobFailed || condition.Type == batchv1.JobComplete {
+			if condition.Type == batchv1.JobFailed {
```

Comment:
please also change restoreManager::makeSyncWarmUpJob() function to reduce the BackoffLimit. @michaelmdeng

Reply:
@BornChanger just updated, thanks!